source/getting-started/user-guide/math-operations.md

```diff
@@ -0,0 +1,78 @@
+# Math Operation Benchmarks
+
+Mathematical operations from the <[math.h](https://pubs.opengroup.org/onlinepubs/009696799/basedefs/math.h.html)> library as well as common operations and casts were timed and benchmarked on the AMDC.
```

Comment:
Looks like an outdated link, please link to:

https://pubs.opengroup.org/onlinepubs/9699919799/